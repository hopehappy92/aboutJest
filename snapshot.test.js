// prettier를 설치해야지 동작함
// test option [-u]로 업데이트 가능

const snapshot = require("./src/snapshot");

test("repeats words three times inline", () => {
  expect(snapshot.repeat("Test", 3)).toMatchInlineSnapshot(`"Test,Test,Test"`);
});

test("repeats words threee times file", () => {
  expect(snapshot.repeat("Test", 3)).toMatchSnapshot();
});
