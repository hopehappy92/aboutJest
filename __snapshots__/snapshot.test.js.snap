// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`repeats words threee times file 1`] = `"Test,Test,Test"`;
